pkg/controller/handler.go

```diff
@@ -4,72 +4,70 @@ import (
 	"github.com/containous/maesh/pkg/k8s"
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
 )
 
+// configMessageChanRebuild rebuild.
+const configMessageChanRebuild = "rebuild"
```

Comment:
We could get rid of this and change the `configRefreshChan` from a `string` into `struct{}`. We are not interested in any information we receive from this chan. WDYT?